cmd/agent/subcommands/run/command.go

```diff
@@ -526,9 +526,17 @@ func startAgent(
 	_ cloudfoundrycontainer.Component,
 	jmxLogger jmxlogger.Component,
 	settings settings.Component,
+	agenttelemetryComponent agenttelemetry.Component,
 ) error {
 	var err error
 
+	span, _ := agenttelemetryComponent.StartStartupSpan("agent.startAgent")
+	span.SetTag("agent_version", version.AgentVersion)
+	span.SetTag("agent_flavor", flavor.GetFlavor())
```

Comment:
I wonder in this case if these tags addition could be done inside of the agenttelemetry component